pkg/netflow/common/flow.go

```diff
@@ -42,8 +43,9 @@ type Flow struct {
 	TCPFlags uint32 `json:"tcp_flags"`
 
 	// Ports for UDP and TCP
-	SrcPort uint32 // FLOW KEY
-	DstPort uint32 // FLOW KEY
+	// Port number can be zero/positive or `-1` (ephemeral port)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
```suggestion
	// Port number can be zero/positive or `*` (ephemeral port)
```